botCommands/simpleCommands.js

```diff
@@ -110,6 +110,12 @@ const justDoIt = {
 };
 registerBotCommand(justDoIt.regex, justDoIt.cb);
 
+const xy = {
+  regex: /(?<!\S)\/xy(?!\S)/,
+  cb: () => 'What problem are you *really* trying to solve? Check out this article to help others better understand your question: https://xyproblem.info/',
```

Comment:
Small change, let’s make sure we remove the Discord embed around the link. See the question command and how we handle the link there.

Reply:
What do you mean by "Discord Embed"?

Reply:
I looked over the question command under simple commands and other than the bold markdown notation, everything else seems to be the same. What exactly should I add?

Reply:
We want arrow brackets around the link: https://support.discord.com/hc/en-us/articles/206342858--How-do-I-disable-auto-embed-

Reply:
I made a commit with the arrow brackets but it failed the test. The question command under simpleCommands.js does not currently have brackets.